setup.py

```diff
@@ -1,23 +1,2 @@
-"""
-Setup file for tubes.
-"""
-
-from setuptools import setup, find_packages
-
-setup(
-    name='Tubes',
-    version='0.2.0',
-    description="""
-    Flow control and backpressure for event-driven applications.
-    """,
-    license="MIT",
-    url="https://github.com/twisted/tubes/",
-
-    packages=find_packages(exclude=[]),
-    package_dir={'tubes': 'tubes'},
-    install_requires=[
-        "six",
-        "Twisted",
-    ],
-    include_package_data=True,
```

Comment:
This seems to be lost. OTOH, it may be unnecessary if there's no package data files to ship.

Reply:
I removed it because there's no package data